proto/build.rs

```diff
@@ -39,6 +39,7 @@ fn main() -> Result<()> {
     config.compile_protos(&["proto/stake.proto"], &["proto/"])?;
     config.compile_protos(&["proto/chain.proto"], &["proto/"])?;
     config.compile_protos(&["proto/genesis.proto"], &["proto/"])?;
+    config.extern_path(".ibc", "::ibc-proto::ibc");
```

Comment:
I think you mentioned on Discord that this is related to some problem with import paths for the `ibc-rs` types, would it be possible to have a comment explaining what it's doing?

Side Q: wait, is `::ibc-proto::ibc` a Rust module path? are dashes allowed there?

Reply:
The codegen apparently converts the dash to an underscore

Reply:
wild!